trade-finance-manager-ui/component-tests/utilisation-reports/_macros/payment-details-table-row.component-test.ts

```diff
@@ -175,6 +197,18 @@ describe(component, () => {
       wrapper.expectElement('tr').toHaveCount(3);
     });
 
+    it('should visually hide all rows other than first row', () => {
+      const paymentDetailsRow: PaymentDetailsRowViewModel = {
+        ...aPaymentDetailsTableRow(),
+        feeRecords: [aFeeRecord(), aFeeRecord(), aFeeRecord()],
+      };
+      const wrapper = getWrapper({ paymentDetailsRow });
+
+      wrapper.expectElement('tr:eq(0)').toHaveAttribute('aria-hidden', 'false');
```

Comment:
Not something to block merging this PR at all so will approve, could something like this be a loop so the test doesn't need to know how many <tr> blocks there will be? 
Wondering if we can loop over everything matching 'tr', from a quick play it looks like we'd have to write our own helper so the answer is probably not easily sadly

Reply:
Yeah I'm not aware of a simple way of achieving this so possibly not worth it, but agree that would be nicer if done in a nice way. I might have a play later if I have some down time.